[2024-09-12T02:17:59.975+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-12T02:17:59.992+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_catchup_backfill_v01.task1 scheduled__2021-10-01T00:00:00+00:00 [queued]>
[2024-09-12T02:18:00.002+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_catchup_backfill_v01.task1 scheduled__2021-10-01T00:00:00+00:00 [queued]>
[2024-09-12T02:18:00.003+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-12T02:18:00.091+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): task1> on 2021-10-01 00:00:00+00:00
[2024-09-12T02:18:00.100+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=3961) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-12T02:18:00.101+0000] {standard_task_runner.py:72} INFO - Started process 3971 to run task
[2024-09-12T02:18:00.100+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_catchup_backfill_v01', 'task1', 'scheduled__2021-10-01T00:00:00+00:00', '--job-id', '19171', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_catchup_and_backfill.py', '--cfg-path', '/tmp/tmp6codn1r5']
[2024-09-12T02:18:00.103+0000] {standard_task_runner.py:105} INFO - Job 19171: Subtask task1
[2024-09-12T02:18:00.194+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_catchup_backfill_v01.task1 scheduled__2021-10-01T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-12T02:18:00.325+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_catchup_backfill_v01' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2021-10-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-10-01T00:00:00+00:00'
[2024-09-12T02:18:00.326+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-12T02:18:00.363+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-12T02:18:00.371+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo This is a simple bash command!']
[2024-09-12T02:18:00.391+0000] {subprocess.py:86} INFO - Output:
[2024-09-12T02:18:00.392+0000] {subprocess.py:93} INFO - This is a simple bash command!
[2024-09-12T02:18:00.393+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-12T02:18:00.433+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-12T02:18:00.435+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_catchup_backfill_v01, task_id=task1, run_id=scheduled__2021-10-01T00:00:00+00:00, execution_date=20211001T000000, start_date=20240912T021759, end_date=20240912T021800
[2024-09-12T02:18:00.520+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-12T02:18:00.533+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
